.docker/btsieve/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM rust:1.37-buster as builder
+
+WORKDIR /src
+COPY ./btsieve ./btsieve
```

Comment:
Instead, use a `.dockerignore` file to exclude `api_tests`, `target`, etc.
This would allow you to do `COPY ./ ./`.
See https://docs.docker.com/engine/reference/builder/

Reply:
I did that first yes but that causes problems with the `btsieve.toml` config file.
I can give it another go :)